mappings/net/minecraft/client/render/block/entity/BlockEntityRenderer.mapping

```diff
@@ -1,6 +1,15 @@
 CLASS net/minecraft/class_827 net/minecraft/client/render/block/entity/BlockEntityRenderer
 	FIELD field_20989 blockEntityRenderDispatcher Lnet/minecraft/class_824;
+	METHOD <init> (Lnet/minecraft/class_824;)V
+		ARG 1 dispatcher
 	METHOD method_23082 getSprite (Lnet/minecraft/class_2960;)Lnet/minecraft/class_1058;
 		ARG 1 id
+	METHOD method_3563 shouldSendToChunkBuilder (Lnet/minecraft/class_2586;)Z
```

Comment:
When is this true? If it's for block entities that render as normal blocks (like the furnace), it could 
 be `rendersAsRegularBlock`.

Reply:
this is true for beacons and what else...

Reply:
What about something like `isBlockModelRendered` (it also renders the regular block as well as custom rendering)?

Reply:
This looks like block entities that render things beyond itself. Fyi this is true for beacon and structure block.

Reply:
Based on the current name, this makes me think that if the method returns true, the tile entity block is sent to the chunk builder to render its block model as part of the chunk. So for example beacons, structure blocks, but not chests.

If that's what it does, `shouldRenderBlock` would be a better name.

Otherwise, the name should be improved to make it clear when I should be overriding this method in my mod.

Reply:
Based on the current name, this makes me think that if the method returns true, the tile entity block is sent to the chunk builder to render its block model as part of the chunk. So for example beacons, structure blocks, but not chests.

If that's what it does, `shouldRenderBlock` would be a better name.

Otherwise, the name should be improved to make it clear when I should be overriding this method in my mod.

Reply:
This is not block model, but it renders a lot of stuff beyond the block model (like beacon or structure block). You should look into it on your own given I am not that good at rendering...

Reply:
@liach Figured out what this does and mapped it in [my PR](https://github.com/FabricMC/yarn/pull/974). When it's true, the chunk builder will add the block entity to a list of block entities that are always rendered, even if they are in a chunk that's culled (for example, underground or behind the player).

This is true for the beacon and the structure block because they render the beam and the wireframe outside of their bounding box, which are visible even if the block itself isn't.